voidcrew/code/modules/mob/living/carbon/human/species_types/kepori.dm

```diff
@@ -53,6 +52,9 @@
 		if(!human.get_bodypart(BODY_ZONE_HEAD))
 			return FALSE
 		return equip_delay_self_check(item, human, bypass_equip_delay_self)
+	if (slot == ITEM_SLOT_BACK)
+		if(item.kepori_equip == FALSE)
```

Comment:
This variable feels pointless, can't they already wear smugglers satchels? This can probably use weightclass, or something, instead of making an entirely new variable

Reply:
i just dont like the idea of placing this variable on EVERY item. You could make a list on the species itself, or i dunno something that doesnt add a variable onto a shitload of uneccesary things. Im not at my pc rn, so ill help you out when i am, if youre still stuck